addons/website/controllers/main.py

```diff
@@ -336,7 +336,17 @@ def website_configurator(self, step=1, **kwargs):
 
     @http.route(['/website/social/<string:social>'], type='http', auth="public", website=True, sitemap=False)
     def social(self, social, **kwargs):
-        url = getattr(request.website, 'social_%s' % social, False)
+        default_values = {
+            'facebook': 'https://www.facebook.com/Odoo',
+            'github': 'https://github.com/odoo',
+            'linkedin': 'https://www.linkedin.com/company/odoo',
+            'youtube': 'https://www.youtube.com/user/OpenERPonline',
+            'instagram': 'https://www.instagram.com/explore/tags/odoo/',
```

Comment:
or https://www.instagram.com/odoo.official ?

Reply:
Makes more sense indeed to me too, I just copy pasted the ones from here
https://github.com/odoo/odoo/blame/1882d8f89f760bd1ff8a2bf0ae798939402647a3/addons/social_media/demo/res_company_demo.xml#L9
I guess 6 years ago we did not have an Odoo instagram account :+1: 
I'll made the change